docs/gettsim_objects/input_variables.rst

```diff
@@ -101,3 +101,27 @@ household.
 +-------------------------+---------------------------------------------+--------------+
 | _`priv_rente_m`         | Amount of monthly private pension           | float        |
 +-------------------------+---------------------------------------------+--------------+
+| _`weiblich`             | True if female                              | bool         |
++-------------------------+---------------------------------------------+--------------+
+| _`pflichtbeitragszeit`  | Months of mandtory contributions            | float        |
++-------------------------+---------------------------------------------+--------------+
+| _`freiw_beitragszeit`   | Months of voluntary contributions           | float        |
++-------------------------+---------------------------------------------+--------------+
+|| _`anrechnungszeit`     || Months of sickness, pregnancy/Mutterschutz,|| float       |
+||                        || unemployment, school/university after age  ||             |
+||                        || 17 (max 8 years), insurance retirement     ||             |
++-------------------------+---------------------------------------------+--------------+
+|| _`anrechnungszeit_45`  || Months of sickness, unemployment (only time|| float       |
```

Comment:
Not sure about the 45 here and 15, 35 below in function names. What if legislator decides to change that to 40, 50 at some point? 

Also, description of the contents should first get a part what these things are used for ("Additional periods considered towards qualifying to retire at `ges_rente_bes_lang_altersgrenze`)" or so.

Reply:
Anrechnungszeiten are not an input anymoe but a function. In general e.g. "Wartezeit x Jahre" is a fixed definition for certain times that are considered (couterintuitively). That is Wartezeit 35 Jahre contians different times than Wartezeit 25 Jahre -  the length is just a side condition. However they can in priciple be applied to different pension concepts, therefore naming them after the pension would make it less transparent and less aligned with the law.